spec/support/i18n.rb

```diff
@@ -0,0 +1 @@
+I18n.load_path += Dir[File.expand_path("../locales/*.yml", __dir__)]
```

Comment:
We definitely shouldn't have test-only code like this. If we need these translations for code to work, then we need them in the main codebase. And if we don't need these translations in the main codebase, then we shouldn't need them here.

It looks like it's just the day_of_month and month_of_year validations that need fixing so that these translations can just be completely removed.